linalg/vector.hpp

```diff
@@ -143,28 +166,39 @@ class Vector
        in addition to the overloaded operator()(int). */
    inline operator const double *() const { return data; }
 
-   inline bool OwnsData() const { return (allocsize > 0); }
+   /// Return a reference to the Memory object used by the Vector.
+   Memory<double> &GetMemory() { return data; }
+
+   /** @brief Return a reference to the Memory object used by the Vector, const
+       version. */
+   const Memory<double> &GetMemory() const { return data; }
+
+   /// Update the memory location of the vector to match @a v.
+   void SyncMemory(const Vector &v) { GetMemory().SyncWith(v.GetMemory()); }
+
```

Comment:
Another short-cut could also be useful, like :
```void SyncAlias(const Vector &v) { GetMemory().SyncAliasToBase(v.GetMemory(),Size()); }```

Reply:
Called it `SyncAliasMemory`